{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/core/pagination.py

```diff
@@ -9,21 +10,24 @@ class LinkedPagination(pagination.PageNumberPagination):
     page_size_query_param = "pageSize"
 
     def get_paginated_response(self, data):
+        url = self.request.build_absolute_uri()
+        first_page = 1
+        last_page = math.floor(self.page.paginator.count / self.page.paginator.per_page) + 1
+        page_count = math.ceil(self.page.paginator.count / self.page.paginator.per_page)
```

Comment:
I could be reading this incorrectly (its been a while since I've done paginator math), but I think the `last_page` gives you one too many if the total divides evenly by the page count. (i.e. total 10, 10 per page, this gives a last page of 2).

Is there effectively a difference between the last page and page count since we're always starting at 1?

Reply:
Thanks for that, the logic there was definitely out of whack.